README.md

```diff
@@ -0,0 +1,189 @@
+| `debug` | Debug mode with extra checks and logging | `false` | `false`, `true` |
+
+## Usage
+
+### Run Once
+
+`run once` runs the provided command. Use `--` to separate `ballot`'s own
+arguments from the command's executable and arguments if they contain `-`'s
+(which is very likely).
+
+Note: If ZooKeeper goes down while a `ballot` process is a leader and others are
+waiting as followers, they will all stay in their respective roles and operate
+until ZooKeeper comes back. They will, however, not be able to perform any
+failover or run new commands until ZooKeeper is back up.
+
+#### Options
+
+| Parameter | Description | Default | Allowed values |
+| --- | --- | --- | --- |
+| `candidate-id` | This `ballot`'s ID, to be used for display and debugging purposes | N/A, mandatory | Any string |
+| `on-child-success` | What to do when the command we run terminates with exit code 0. | `reelect` | `rerun`, `reelect`, `exit`, `ignore` |
+| `on-child-error` | What to do when the command we run terminates with an exit code other than 0, or cannot be executed for any reason. | `reelect` | `rerun`, `reelect`, `exit`, `ignore` |
+| `on-election-failure` | What to do when the election process fails for technical reasons (unreachable ZooKeeper, for example). Use `run-anyway` only if running too many processes at once is preferable to none at all. | `retry` | `retry`, `run-anyway`, `exit` |
+| `wrap-child-logs` | Whether to display the command's logs formatted as our own logs, or have its `stdout` and `stderr` directly connected to `ballot`'s | `true` | `false`, `true` |
+
+### Run Cron
+
+`run cron` runs the provided command on a schedule. Use `--` to separate `ballot`'s own
+arguments from the command's executable and arguments if they contain `-`'s
+(which is very likely).
+
+Leader election is performed once and all timed invocations happen on the node
+that was elected leader. Elections do not happen for each invocation, unless
+`on-child-success`/`on-child-error` are set to `reelect`.
+
+The only supported concurrency policy at the moment is to skip an invocation if
+the previous invocation is still running.
+
+Note: If ZooKeeper goes down while a `ballot` process is a leader and others are
+waiting as followers, they will all stay in their respective roles and operate
+until ZooKeeper comes back. They will, however, not be able to perform any
+failover in the event that the leader goes down.
+
+Future features can include adding a deadline and killing overdue jobs, allowing
+invocation queueing, allowing concurrent invocations.
+
+#### Options
+
+| Parameter | Description | Default | Allowed values |
+| --- | --- | --- | --- |
+| `schedule` | Schedule to run on. | N/A, mandatory | [Extended Cron format](https://pkg.go.dev/github.com/robfig/cron#hdr-CRON_Expression_Format) |
+| `candidate-id` | This `ballot`'s ID, to be used for display and debugging purposes | N/A, mandatory | Any string |
```

Comment:
Maybe specify that this needs to be unique among the various ballots (I'm assuming it needs to be)

Reply:
It doesn't need to be unique since it's for informative only, and verifying that it's unique wouldn't be trivial. But I agree it's worth a mention that inspecting would be more difficult if it's not unique.